src/App.js

```diff
@@ -0,0 +1,52 @@
+import './App.css';
+import Header from './components/Header/Header';
+import Status from './components/Status/Status';
+import { useEffect, useState } from 'react';
+import active from './assets/rocketActive.svg';
+import destroyed from './assets/rocketDeactivate.svg';
+import unknown from './assets/rocketUnknown.svg';
+import Rocket from './components/Rocket/Rocket';
+import Footer from './components/Footer/Footer';
```

Comment:
```suggestion
import { useEffect, useState } from 'react';

import './App.css';

import Header from './components/Header/Header';
import Rocket from './components/Rocket/Rocket';
import Status from './components/Status/Status';
 import Footer from './components/Footer/Footer';

import active from './assets/rocketActive.svg';
import destroyed from './assets/rocketDeactivate.svg';
import unknown from './assets/rocketUnknown.svg';
```
Son de los pequenos detalles que permiten hacer mas legible la lectura del código.

Reply:
Excelente observación! Sinceramente no se me había ocurrido separar los imports por tipo o: